src/DynamoCore/Graph/Connectors/ConnectorModel.cs

```diff
@@ -41,6 +43,28 @@ public class ConnectorModel : ModelBase
         /// </summary>
         public PortModel End { get; private set; }
 
+        /// <summary>
+        /// Collection of ConnectorPinModels associated with this Connector.
+        /// </summary>
+        public ObservableCollection<ConnectorPinModel> ConnectorPinModels { get; private set; }
+
+        /// <summary>
+        /// Handles adding a new ConnectorPin to the collection of CollectorPins associated with this connector.
+        /// This function is called after deserializing and is responsible for reconstructing the pins corresponding
+        /// to each connector.
+        /// </summary>
+        /// <param name="x"></param>
+        /// <param name="y"></param>
```

Comment:
Would be best if we can fill param description as well

Reply:
I guess this is a global comment?

Reply:
@QilongTang